tests/test_trytond_factories.py

```diff
@@ -0,0 +1,70 @@
+
+import unittest
+
+from trytond.tests.test_tryton import DB_NAME
+from trytond.tests.test_tryton import db_exist
+from trytond.tests.test_tryton import create_db
+from trytond.tests.test_tryton import restore_db_cache
+from trytond.tests.test_tryton import backup_db_cache
+from trytond.tests.test_tryton import drop_db
+from trytond.tests.test_tryton import with_transaction
+from trytond.transaction import Transaction
+from trytond.pool import Pool
+
+from factories.company import Company
```

Comment:
`factory` will contain the code from factory-boy, `factory_trytond` will contain the Tryton integration for factory-boy but `factories` could be almost anything.

I'd expect the namespace for this library to be `trytond_factories` instead. Just as the package name but replacing dashes by underscores.

Reply:
Note the Readme has a code fragment featuring the namespace I suggest!

Reply:
I think that you are considering the problem with hoard *factories* keyword assuming that some applications which actually use this package need to declare custom factories as well.

It will be a collision with:
`import factories
from route_to_my_own_factories import factories`

Your suggestion looks good to me!